Lib/test/test_clinic.py

```diff
@@ -1582,7 +1579,8 @@ def test_indent_stack_no_tabs(self):
                *vararg1: object
             \t*vararg2: object
         """
-        err = "Tab characters are illegal in the Clinic DSL."
+        err = ("Tab characters are illegal in the Clinic DSL: "
+               r"'\t*vararg2: object'")
```

Comment:
Tiny nitpick: don't think this needs to be a rawstring, does it?

```suggestion
               "'\t*vararg2: object'")
```

Reply:
No, it's needed because we don't want it to expand. Either we keep the raw string, or we'll have to excape the backslash.

Reply:
Ohh, because it's eventually passed to `assertRaisesRegex`?

Reply:
Probably; it got expanded anyways, so the the test failed.